development/prow-installer/before-commit.sh

```diff
@@ -4,6 +4,8 @@ readonly CI_FLAG=ci
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
+# ensure $GOPATH/bin is present in PATH
+export PATH=$GOPATH/bin:$PATH
```

Comment:
This shouldn't be part of the script. The docker image itself is taking care of this. We should maybe figure out why it is not exported correctly. Are we using the correct image? See
https://github.com/kyma-project/test-infra/blob/master/prow/jobs/test-infra/validation.yaml#L87
and 
https://github.com/kyma-project/test-infra/blob/master/prow/images/buildpack-golang/Dockerfile#L19

Reply:
The image sets up the gopath variable to /workspace/go, as it should be intended. It changes when the image is used in a job. Might be due to some kind of preset. Will check that later.

Reply:
The above line is just to ensure that current gopath/bin is set in path as a workaround.